### PR TITLE
add: `exiftool`

### PR DESCRIPTION
**Pull request by crypt0-wizard**
Add ExifTool package to Pacstall repository

ExifTool is a platform-independent Perl library and command-line tool for reading, writing, and editing metadata in a wide variety of files. This PR adds:
- exiftool.pacscript: Package script for version 13.03
- .SRCINFO: Package metadata file

Features:
- Installs the latest stable version (13.03)
- Includes documentation files
- Properly handles dependencies (perl)
- Uses standard build process with perl's Makefile.PL

Testing:
- Package builds and installs successfully
- Binary runs correctly after installation

### CODE REVIEW

File: packages/exiftool/exiftool.pacscript [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name="exiftool"
+pkgname="exiftool"
+pkgver="13.03"
+pkgrel="1"
```
Comment (vigress8) [inline packages/exiftool/exiftool.pacscript:4]:
```suggestion
```
Redundant

File: packages/exiftool/exiftool.pacscript [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name="exiftool"
```
Comment (vigress8) [inline packages/exiftool/exiftool.pacscript:1]:
```suggestion
```
This variable is not used anymore

File: packages/exiftool/exiftool.pacscript [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+pkgname="exiftool"
+pkgver="13.03"
+pkgrel="1"
+pkgdesc="A platform-independent Perl library for reading and writing EXIF data"
+maintainer="Ishaque <muhammadishaque812@gmail.com>"
```
Comment (vigress8) [inline packages/exiftool/exiftool.pacscript:6]:
```suggestion
maintainer=("Ishaque <muhammadishaque812@gmail.com>")
```
`maintainer` is an array

File: packages/exiftool/exiftool.pacscript [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+url="https://exiftool.org"
+arch=('any')
+license="GPL-2.0-or-later"
```
Comment (vigress8) [inline packages/exiftool/exiftool.pacscript:9]:
```suggestion
license=("GPL-2.0-or-later")
```
`license` is an array

File: packages/exiftool/exiftool.pacscript [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+depends=('perl')
+gives="exiftool"
+
+source=("https://exiftool.org/Image-ExifTool-${pkgver}.tar.gz")
+sha256sums=('0912e1315318889574f355e5832340632c556a14d30711e94d801085ad0a8e4f')
+
+prepare() {
+  cd "Image-ExifTool-${pkgver}"
+}
+
```
Comment (vigress8) [inline packages/exiftool/exiftool.pacscript:19]:
```suggestion
```
Redundant

File: packages/exiftool/exiftool.pacscript [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  make
+}
+
+package() {
+  cd "Image-ExifTool-${pkgver}"
+  make DESTDIR="${pkgdir}" install
+
+  # Install documentation
+  install -Dm644 README "${pkgdir}/usr/share/doc/${name}/README"
+  install -Dm644 Changes "${pkgdir}/usr/share/doc/${name}/Changes"
```
Comment (vigress8) [inline packages/exiftool/exiftool.pacscript:32]:
```suggestion
  install -Dm644 README Changes -t "${pkgdir}/usr/share/doc/${pkgname}"
```

**Review by vigress8 — CHANGES_REQUESTED** [review]

File: packages/exiftool/exiftool.pacscript [hunk 7]
```diff
@@ -0,0 +1,33 @@
+name="exiftool"
+pkgname="exiftool"
+pkgver="13.03"
+pkgrel="1"
+pkgdesc="A platform-independent Perl library for reading and writing EXIF data"
```
Comment (vigress8) [inline packages/exiftool/exiftool.pacscript:5]:
```suggestion
pkgdesc="Platform-independent Perl library for reading and writing EXIF data"
```

**Comment by vigress8** [comment]
Please enable "Allow edits by maintainers", otherwise the pre-commit CI bot can't do its job

**Comment by crypt0-wizard** [comment]
> Please enable "Allow edits by maintainers", otherwise the pre-commit CI bot can't do its job

**Comment by crypt0-wizard** [comment]
> > Please enable "Allow edits by maintainers", otherwise the pre-commit CI bot can't do its job

Done

**Review by vigress8 — APPROVED** [review]

File: packages/exiftool/exiftool.pacscript [hunk 8]
```diff
@@ -0,0 +1,27 @@
+pkgname="exiftool"
+pkgver="13.03"
+pkgdesc="Platform-independent Perl library for reading and writing EXIF data"
+maintainer=("Ishaque <muhammadishaque812@gmail.com>")
+url="https://exiftool.org"
```
Comment (Elsie19) [inline packages/exiftool/exiftool.pacscript:5]:
```suggestion
url='https://exiftool.org'
```

File: packages/exiftool/exiftool.pacscript [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+arch=('any')
+license=("GPL-2.0-or-later")
+depends=('perl')
+gives="exiftool"
+repology=("project: ${gives}")
```
Comment (Elsie19) [inline packages/exiftool/exiftool.pacscript:10]:
```suggestion
repology=("project: ${pkgname}")
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/exiftool/exiftool.pacscript [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+depends=('perl')
+repology=("project: ${pkgname}")
+
+source=("https://exiftool.org/Image-ExifTool-${pkgver}.tar.gz")
+sha256sums=('0912e1315318889574f355e5832340632c556a14d30711e94d801085ad0a8e4f')
+
+build() {
+  cd "Image-ExifTool-${pkgver}"
+  perl Makefile.PL
+  make
```
Comment (Elsie19) [inline packages/exiftool/exiftool.pacscript:17]:
```suggestion
  make -j"${NCPU}"
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by vigress8 — DISMISSED** [review]

File: packages/exiftool/exiftool.pacscript [hunk 11]
```diff
@@ -0,0 +1,26 @@
+pkgname="exiftool"
+pkgver="13.03"
+pkgdesc="Platform-independent Perl library for reading and writing EXIF data"
+maintainer=("Ishaque <muhammadishaque812@gmail.com>")
+url='https://exiftool.org'
+arch=('any')
+license=("GPL-2.0-or-later")
+depends=('perl')
+repology=("project: ${pkgname}")
+
```
Comment (Elsie19) [inline packages/exiftool/exiftool.pacscript:10]:
```suggestion
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**